src/lib.jl

```diff
@@ -1,85 +1,60 @@
 using CEnum
 
-const __time_t = Clong
+const __darwin_time_t = Clong
```

Comment:
@Octogonapus, I ran the generator.jl script on my macos; do you know if there's a setting to avoid the __darwin prefix here? Or maybe we need that and only load it on apple? It'd be nice to have all the common definitions in one place, then anything platform-specific (if necessary) in their own smaller files w/ just the platform-specific stuff.

Reply:
> do you know if there's a setting to avoid the __darwin prefix here? Or maybe we need that and only load it on apple

I don't know if there is such a setting and I don't know if it's needed on apple. It looks like we don't need that and can just use the bindings generated on linux.

> anything platform-specific (if necessary) in their own smaller files w/ just the platform-specific stuff

I don't think this can be done. We should really be running the generator in CI in a repeatable way.

Reply:
#6 is my take on doing this automatically. We should also be able to run this with a cron trigger and regenerate bindings when the JLL changes.